ertviz/controllers/parameter_controller.py

```diff
@@ -1,11 +1,12 @@
 import re
+from tests import data
```

Comment:
should this be here?

Reply:
yes, that's a legacy from before. Also `import json` can go away